cli/tests/data/marathon/pods/updated_good.json

```diff
@@ -15,5 +15,10 @@
         "mem": 16.0
       }
     }
+  ],
+  "networks": [
```

Comment:
You added a new commit that added just the networks field, so my question is when did you realize this field was necessary? If you realized it because existing tests failed, than no need to add a new one imo.